v2/si/security-insights.go

```diff
@@ -57,6 +57,7 @@ type Project struct {
 	Funding        string     `yaml:"funding"`
 	Administrators []Contact  `yaml:"administrators"`
 	Repositories   []Repo     `yaml:"repositories"`
+	Steward        *Link      `yaml:"steward"`
```

Comment:
Unfortunately when https://github.com/ossf/security-insights-spec/commit/4c4a6319b3b302f524e6d6ba01f0676278ac54ed merged, a strict YAML parse/decode (i.e. one with reject unknown fields enabled) would fail when attempting to decode any insights file like https://github.com/ossf/security-insights-spec/blob/main/.github/security-insights.yml that had a `project.steward` defined.